taxi/urls.py

```diff
@@ -7,6 +7,9 @@
     DriverListView,
     DriverDetailView,
     ManufacturerListView,
+    CarCreateView,
+    ManufacturerCreateView, CarUpdateView, ManufacturerUpdateView,
+    CarDeleteView, ManufacturerDeleteView,
```

Comment:
Here you need to strict to one style: keep one class in one line if it is what was done before.